lupa/_lupa.pyx

```diff
@@ -1719,14 +1757,21 @@ cdef build_lua_error_message(LuaRuntime runtime, lua_State* L, int stack_index=-
 
 # calling into Lua
 
-cdef run_lua(LuaRuntime runtime, bytes lua_code, tuple args):
+cdef run_lua(LuaRuntime runtime, bytes lua_code, bytes name, bytes mode, tuple args):
     """Run Lua code with arguments"""
     cdef lua_State* L = runtime._state
+    cdef const char* c_name = b'<python>'
```

Comment:
This line I would change to

```suggestion
    cdef const char* c_name = b'=<python>'
```
or

```suggestion
    cdef const char* c_name = NULL
```
for the reason explained [here](https://github.com/scoder/lupa/issues/248#issuecomment-1822909833).

Reply:
I've seen your comment but it would introduce an incompatible change. It would break doctests, for example, or code that tries to parse Lua errors.